spec/support/shared_examples/request_method.rb

```diff
@@ -44,10 +44,29 @@
   #   end
   # end
 
+  on_feature :body_on_get do
```

Comment:
nit: This now is tested for all verbs, should we rename the feature to something more generic?

Reply:
How about `:request_body_on_query_methods`?

Reply:
Sounds good!